tag-semantic-release/action.yaml

```
@@ -35,9 +35,8 @@ runs:
           exit 1
         fi
 
-        # Last 5 recent tags
         git fetch --tags --quiet
-        tags=$(git for-each-ref refs/tags --sort=-creatordate --format="%(refname:lstrip=2)" --count=5)
+        tags=$(git tag | sort --version-sort -r)
```

I'd suggest using long opts for both of these

```suggestion
        tags=$(git tag | sort --reverse --version-sort)
```